standards/SPIFFE-ID.md

```diff
@@ -45,6 +46,15 @@ Trust domains are nominally self-registered, unlike public DNS there is no deleg
 
 The trust domain is defined as the authority component of the URI - specifically, the `host` part of the authority. The `userinfo` and `port` parts of the authority component MUST NOT be set, and the `:` delimiter MUST NOT be present. Please see section 3.2 of [RFC 3986](https://tools.ietf.org/html/rfc3986) for more information.
 
+#### 2.1.1. Trust Domain Name Collisions
+
+Trust domain names are self-registered, ie. trust domain operators are free to choose any trust domain name they find suitable. Since there is no centralized authority for regulation or registration of trust domain names, there is no guarantee of global uniqueness nor is there any technical means for preventing distinct trust domains from using identical trust domain names.
+
+To prevent accidental collisions, operators are advised to select trust domain names which are highly likely to be globally unique. When available, using a registered domain name as a suffix of a trust domain name will reduce chances of an accidental collision; for example, if a trust domain operator owns the domain name `example.com`, then using a trust domain name such as `trust_domain_name.example.com` would be reasonable. When trust domain names are automatically generated without operator input, random generating a GUID is strongly advised.
+
+When a collision does occur (ie. two trust domains select identical names), those trust domains will continue to operate independently but will be unable to federate (ie. connect to one and other). Collisions, however, pose no impact to the security of a SPIFFE-compliant trust domain system: SPIFFE-based authenticatation relies on a (cryptographic) mechanism independent of a trust domain name. When performing SPIFFE-based authentication, the public key(s) of the root of a trust domain are used to validate an identity. Further details of SPIFFE authentication are covered in [Section 3.1](#31-svid-trust).
```

Comment:
```suggestion
When a collision does occur (ie. two trust domains select identical names), those trust domains will continue to operate independently but will be unable to federate (ie. connect to one and other). Collisions, however, pose no impact to the security of a SPIFFE-compliant trust domain system: identity claims issued by one trust domain will fail validation in the other and vice versa. Further details of SPIFFE authentication are covered in [Section 3.1](#31-svid-trust).
```

I feel like it's not completely accurate to say that authentication is "independent" of the trust domain name, since the trust domain name is used as a key to look up the correct trust bundle.  My suggestion is to skip the details entirely and say they'll be covered in a later section.

Reply:
I scrubbed most of the details. PTAL.